waspc/data/Generator/templates/server/src/auth/providers/oauth/redirect.ts

```diff
@@ -0,0 +1,19 @@
+{{={= =}=}}
+import { config } from 'wasp/server'
+
+export const loginPath = '{= serverOAuthLoginHandlerPath =}'
+export const callbackPath = '{= serverOAuthCallbackHandlerPath =}'
+export const exchangeCodeForTokenPath = '{= serverExchangeCodeForTokenHandlerPath =}'
+
+export function getRedirectUriForCallback(providerName: string) {
+  return `${config.serverUrl}/auth/${providerName}/${callbackPath}`;
+}
+
+const clientOAuthCallbackPath = '{= clientOAuthCallbackPath =}'
```

Comment:
I'd move this to the end of the file (or at least add a space), or even the beginning next to other constants.